Editor/Utility/CinemachineEditorAnalytics.cs

```diff
@@ -0,0 +1,257 @@
+        {
+            EditorApplication.playModeStateChanged += SendAnalyticsOnPlayEnter;
+        }
+
+        /// <summary>
+        /// Send analytics event when using Create -> Cinemachine menu
+        /// </summary>
+        /// <param name="name">Name of the vcam created</param>
+        public static void SendCreateEvent(string name)
+        {
+            if (!EditorAnalytics.enabled)
+                return;
+
+            var data = new CreateEventData
+            {
+                vcam_created = name,
+            };
+
+            // Register our event
+            EditorAnalytics.RegisterEventWithLimit("cm_create_vcam", 
+                k_MaxEventsPerHour, k_MaxNumberOfElements, k_VendorKey);
+
+            // Send the data to the database
+            EditorAnalytics.SendEventWithLimit("cm_create_vcam", data);
+        }
+
+        struct CreateEventData
+        {
+            public string vcam_created; // vcam created from Create -> Cinemachine menu
+        }
+
+        /// <summary>
+        /// Send analytics event when using entering playmode
+        /// </summary>
+        /// <param name="state">State change to detect entering playmode</param>
+        static void SendAnalyticsOnPlayEnter(PlayModeStateChange state)
+        {
+            // Only send analytics if it is enabled
+            if (!EditorAnalytics.enabled)
+                return;
+
+            // Only send data when entering playmode
+            if (state != PlayModeStateChange.EnteredPlayMode)
+                return;
+
+            var startTime = Time.realtimeSinceStartup;
+
+            var cinemachineCore = CinemachineCore.Instance;
+            var vcamCount = cinemachineCore.VirtualCameraCount;
+            var vcamDatas = new List<VcamData>();
+
+            // collect data from all vcams
+            for (int i = 0; i < vcamCount; ++i)
+            {
+                var vcamBase = cinemachineCore.GetVirtualCamera(i);
+                CollectData(vcamBase, i.ToString(), ref vcamDatas);
+            }
+
+            var projectData = new ProjectData
+            {
+                brain_count = cinemachineCore.BrainCount,
+                vcam_count = cinemachineCore.VirtualCameraCount,
+                cam_count = Camera.allCamerasCount,
+                vcams = vcamDatas,
+                time_elapsed = Time.realtimeSinceStartup - startTime,
+            };
+
+            // Register our event
+            EditorAnalytics.RegisterEventWithLimit("cm_vcams_on_play", 
+                k_MaxEventsPerHour, k_MaxNumberOfElements, k_VendorKey);
+
+            // Send the data to the database
+            EditorAnalytics.SendEventWithLimit("cm_vcams_on_play", projectData);
+        }
+
+        static void CollectData(CinemachineVirtualCameraBase vcamBase, string id, ref List<VcamData> vcamDatas)
+        {
+            if (vcamBase == null) return;
+
+            var vcam = vcamBase as CinemachineVirtualCamera;
```

Comment:
I just thought of this: you also need to support CinemachineNewVirtualCamera, which is a simplified alternative to CinemachineVirtualCamera (no secret child objects).  There is also CinemachineNewFreeLook - which inherits from CinemachineNewVirtualCamera so will not need special treatment.  Both of those are available in the presence of a define.  You'll need conditional code here on that same define

Reply:
! 👍